bin/describe/describe_common.mli

```diff
@@ -0,0 +1,59 @@
+open Import
+
+val sanitize_for_tests : bool ref
+
+type options =
+  { with_deps : bool
+  ; with_pps : bool
+  }
+
+module Descr : sig
+  module Item : sig
+    type t
+  end
+
+  module Workspace : sig
+    type t = Item.t list
+
+    val to_dyn : options -> t -> Dyn.t
+  end
+end
+
+module Crawl : sig
```

Comment:
I don't understand this refactoring very well. The name is `Describe_common` and yet basically everything it contains is specific to a particular subcommand. What's the motivation for this module?

Reply:
Yeah, I didn't bother untangling it further yesterday. I'll try to do that today. There are parts used by multiple commands which I will try to leave here whilst moving other things elsewhere.